### PR TITLE
17 update ui methods

### LINKED ISSUE #17: Move UI stuff from TaskList to Command

### PR DESCRIPTION
**Pull request by alstontham**
Updated the printTaskAdded() and printTaskEdited() methods

### CODE REVIEW

**Review by amanda-chua — APPROVED** [review]
Good

File: src/main/java/athena/Ui.java [hunk 1]
```diff
@@ -23,11 +23,21 @@ public void printWelcomeMessage() {
...
+     * @param taskImportance What is the importance of the task
+     * @param taskNotes Any additional notes the user has added to the task
      */
-    public void printTaskAdded(Task task) {
-        System.out.println("You've successfully added " + task + "!\n"
-                + "Another mission to complete? Let's do it!\n");
+    public void printTaskAdded(String taskName, String taskStartTime, String taskDuration, String taskDeadline,
+                               String taskRecurrence, String taskImportance, String taskNotes) {
+        System.out.println("You've successfully added " + taskName + " to your list!\n"
+                + "This task will start at " + taskStartTime + ", last for " + taskDuration + ", and end at " + taskDeadline + ".\n"
```
Comment (daniellimws) [inline src/main/java/athena/Ui.java]:
```
Error: eckstyle] [ERROR] /home/runner/work/tp/tp/src/main/java/athena/Ui.java:37: Line is longer than 120 characters (found 132). [LineLength]
```

File: src/main/java/athena/Ui.java [hunk 2]
```diff
@@ -23,11 +23,21 @@ public void printWelcomeMessage() {
...
+                + "It is set to happen " + taskRecurrence + " and has an importance of " + taskImportance + ".\n"
+                + "Additionally, you've also added these additional notes!\n" + taskNotes +".\n"
```
Comment (daniellimws) [inline src/main/java/athena/Ui.java]:
```
Error: eckstyle] [ERROR] /home/runner/work/tp/tp/src/main/java/athena/Ui.java:39:91: WhitespaceAround: '+' is not followed by whitespace. Empty blocks may only be represented as {} when not part of a multi-block statement (4.1.3) [WhitespaceAround]
```

File: src/main/java/athena/Ui.java [hunk 3]
```diff
@@ -51,11 +61,27 @@ public void printTaskDone(Task task) {
     }
 
     /**
-     * Prints out a message verifying that the task the user specified has been edited.
-     * @param task The task that the user has edited.
+     * Prints out a message verifying that the task the user specified has been edited, and shows the user the
```
Comment (daniellimws) [inline src/main/java/athena/Ui.java:65]:
```
Error: eckstyle] [ERROR] /home/runner/work/tp/tp/src/main/java/athena/Ui.java:63: First sentence of Javadoc is missing an ending period. [SummaryJavadoc]
```

File: src/main/java/athena/Ui.java [hunk 4]
```diff
@@ -51,11 +61,27 @@ public void printTaskDone(Task task) {
...
+     * @param taskStartTime When the task is scheduled to start
+     * @param taskDuration How long the task will last for
+     * @param taskDeadline When the task is due
+     * @param taskRecurrence When will the task repeat
+     * @param taskImportance What is the importance of the task
+     * @param taskNotes Any additional notes the user has added to the task
      */
-    public void printTaskEdited(Task task) {
-        System.out.println("You've changed the details of the task " + task + "?\n"
+    public void printTaskEdited(int taskIndex, String taskName, String taskStartTime, String taskDuration, String taskDeadline,
```
Comment (daniellimws) [inline src/main/java/athena/Ui.java]:
```
Error: eckstyle] [ERROR] /home/runner/work/tp/tp/src/main/java/athena/Ui.java:75: Line is longer than 120 characters (found 127). [LineLength]
```

**Review by daniellimws — CHANGES_REQUESTED** [review]
Looks good. Have some checkstyle errors.

And one more that is not part of this PR but should be a quick fix.

```
Error: eckstyle] [ERROR] /home/runner/work/tp/tp/src/main/java/athena/Ui.java:111: Line is longer than 120 characters (found 127). [LineLength]
```

**Review by daniellimws — APPROVED** [review]

**Pull request merged**